condor/condor_meter

```diff
@@ -551,11 +551,14 @@ def determine_host_description(classad):
 
     If there's no special host description, this returns None.
     """
-    if 'MATCH_EXP_JOBGLIDEIN_ResourceName' not in classad:
+    try:
+        if classad['MATCH_EXP_JOBGLIDEIN_ResourceName'] == 'Local Job':
+            host_descr = GratiaCore.Config.get_SiteName()
```

Comment:
To be clearer: this change makes sure local job records have the right data. `SuppressGridLocalJobs` decides whether or not to upload the local job records.